pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "fennel-ai"
-version = "1.5.58"
+version = "1.6.0"
```

Comment:
I think 1.5.59 was reverted so you can just use that? Also, why increment the middle version? Isn't this a completely backward compatible change?

Reply:
even if 1.5.59, was publish to 1.5.59 stopped if not then that's a version we cannot remove nor edit. I will change this to 1.5.60.